queries.sql

```diff
@@ -1,3 +1,30 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon"
+SELECT * from animals WHERE name LIKE '%mon%';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Here you have to select all animals born between 2016 and 2019, that means from the first day of 2016 up to the last day of 2019. your code should look like this:

  ```sql
  SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
  ```

  OR

  ```sql
  SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
  ```